docker/testpmd/Dockerfile

```diff
@@ -1,5 +1,4 @@
-FROM ubuntu:22.04@sha256:bbf3d1baa208b7649d1d0264ef7d522e1dc0deeeaaf6085bf8e4618867f03494
-COPY ./requirements.txt /
```

Comment:
meson is no longer required?

Reply:
The file always had the apt install and I had experimented building the dockerfile locally with removing either the apt install one or the pip install one, and it errored the build on pip install, I can get the output there now

Reply:
Command tried to run:
```
  3 | >>> RUN apt-get -q update \
  4 | >>>     && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
  5 | >>>         git \
  6 | >>>         vim \
  7 | >>>         python3-pyelftools \
  8 | >>>         libnuma-dev \
  9 | >>>         python3-pip \
10 | >>>         ninja-build \
11 | >>>         build-essential \
12 | >>>         && pip3 install --no-cache-dir --upgrade --require-hashes -r requirements.txt \
13 | >>>     && rm -rf /var/lib/apt/lists/*

```
Output:
```
...
110.8 × This environment is externally managed
110.8 ╰─> To install Python packages system-wide, try apt install
110.8     python3-xyz, where xyz is the package you are trying to
110.8     install.
110.8
110.8     If you wish to install a non-Debian-packaged Python package,
110.8     create a virtual environment using python3 -m venv path/to/venv.
110.8     Then use path/to/venv/bin/python and path/to/venv/bin/pip. Make
110.8     sure you have python3-full installed.
110.8
110.8     If you wish to install a non-Debian packaged Python application,
110.8     it may be easiest to use pipx install xyz, which will manage a
110.8     virtual environment for you. Make sure you have pipx installed.
110.8
110.8     See /usr/share/doc/python3.12/README.venv for more information.
110.8
110.8 note: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by passing --break-system-packages.
110.8 hint: See PEP 668 for the detailed specification.
```